src/App.tsx

```diff
@@ -3,33 +3,52 @@ import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
-type State = {};
+type State = {
+  searchWord: string,
+};
 
 export class App extends React.Component<{}, State> {
-  state: State = {};
+  state: State = {
+    searchWord: '',
+  };
+
+  filteredMovies = () => {
+    const { searchWord } = this.state;
+
+    return moviesFromServer.filter(
+      movie => movie.title.toLowerCase().includes(searchWord.toLocaleLowerCase())
+      || movie.description.toLowerCase().includes(searchWord.toLocaleLowerCase()),
+    );
```

Comment:
You can optimize this
```suggestion
    const lowerCaseSearchWord = this.state.searchWord.toLocaleLowerCase();

    return moviesFromServer.filter(
      movie => movie.title.toLowerCase().includes(lowerCaseSearchWord)
      || movie.description.toLowerCase().includes(lowerCaseSearchWord),
    );
```